### DISCUSSION TITLE
When using 2 GRADKIENT_POLYLINE elements, only the first one works

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#142] [Developer Feedback]
I tried to draw 2 graphs using GRADKIENT_POLYLINE elements. But only the first element is displayed. I tried swapping them around to make sure the error wasn't in the elements. Still showing only the first element.

**Comment by MyoungXUE** [comment]
Thank you for your feedback, I will feedback this issue to the relevant colleagues.

**Comment by MyoungXUE** [comment]
Hello, this widget does not support the creation of multiple, this problem our colleagues have been in the optimization.
